tests/logging/jobWrapperLogFuncs.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/logging/jobWrapperLogFuncs.js
+ */
+'use strict'; // eslint-disable-line strict
+
+const expect = require('chai').expect;
+const tu = require('../testUtils');
+const u = require('./utils');
+const Aspect = tu.db.Aspect;
+const path = '/v1/samples/upsert/bulk';
+const jobWrapper = require('../../jobQueue/jobWrapper');
+const jobQueue = require('../../jobQueue/jobWrapper').jobQueue;
+const sinon = require('sinon');
+
+describe('jobWrapper: functions ' + path, () => {
+  let token;
+
+  before((done) => {
+    tu.toggleOverride('useWorkerProcess', true);
+    tu.toggleOverride('enableWorkerActivityLogs', true);
+    tu.createToken()
+    .then((returnedToken) => {
+      token = returnedToken;
+      done();
+    })
+    .catch(done);
+  });
+
+  before((done) => {
+    Aspect.create({
```

Comment:
Good, we don't need it. I created this while playing with the test and forgot to remove. Let me remove it in my next PR for clock logs.